src/backend/distributed/commands/dependencies.c

```diff
@@ -681,6 +719,26 @@ ShouldPropagateObject(const ObjectAddress *address)
 }
 
 
+/*
+ * ShouldPropagateAnyObject determines if we should be propagating DDLs based
+ * on their object addresses.
+ */
+bool
+ShouldPropagateAnyObject(List *addresses)
+{
+	ObjectAddress *address = NULL;
+	foreach_ptr(address, addresses)
+	{
+		if (!ShouldPropagateObject(address))
```

Comment:
should we return true `if(ShouldPropagateObject)` ? `ShouldPropagateAnyObject` implies that, similar to IsAnyObjectDistributed